integration/bundle/init_test.go

```diff
@@ -39,6 +40,13 @@ func TestBundleInitErrorOnUnknownFields(t *testing.T) {
 //     make changes that can break the MLOps Stacks DAB. In which case we should
 //     skip this test until the MLOps Stacks DAB is updated to work again.
 func TestBundleInitOnMlopsStacks(t *testing.T) {
+	nineJan2025 := time.Date(2025, time.January, 9, 0, 0, 0, 0, time.UTC)
+	oneJan2025 := time.Date(2025, time.January, 1, 0, 0, 0, 0, time.UTC)
+	if time.Now().Before(nineJan2025) && time.Now().After(oneJan2025) {
```

Comment:
1. Why check `time.Now()` is after today?
2. I like the idea of skipping a test until some time in the future to prevent forgetting about it. Could you add a helper function to `testutil` for this, e.g. `testutil.SkipUntil`.

Reply:
> Why check time.Now() is after today?

I wanted to hedge against the Github VM runner's clocks being broken and then us permanently skipping these tests. This way we are confident the clock likely is working.

Reply:
If that's not a concern then I'll add a SkipUntil function, otherwise I'll add a SkipBetween function.

Reply:
I think clock being broken is not a concern but it does add extra complexity and requires user to provide extra parameter. So I'd go with a single date in the future.

Reply:
Fair enough, added the SkipUntil function.